bigtop-manager-server/src/main/resources/ddl/MySQL-DDL-CREATE.sql

```diff
@@ -321,6 +321,66 @@ CREATE TABLE `stage`
     KEY              `idx_job_id` (`job_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+CREATE TABLE `llm_platform`
+(
+    `id`             BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `name`           VARCHAR(255)        NOT NULL,
+    `api_url`        VARCHAR(255)        NOT NULL,
```

Comment:
Can we remove `api_url` column and use the constant defined in `AIAssistant` class?